Criterion.hs

```diff
@@ -41,9 +41,9 @@ import Control.Monad (void)
 import Criterion.IO.Printf (note)
 import Criterion.Internal (runAndAnalyseOne)
 import Criterion.Main.Options (defaultConfig)
-import Criterion.Measurement (initializeTime)
 import Criterion.Monad (withConfig)
 import Criterion.Types
+import Criterion.Measurement (initializeTime)
```

Comment:
Move this back to make the diff less noisy.

Reply:
What do you mean? Where should I move this?

Reply:
This PR needlessly moves this import two lines down. Just put it back to where it originally was (as shown in the diff).